docs/reference/index-modules/blocks.asciidoc

```diff
@@ -76,7 +76,7 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index]
 +
 To add blocks to all indices, use `_all` or `*`. To disallow the adding
 of blocks to indices with `_all` or wildcard expressions,
-change the `action.destructive_requires_name` cluster setting to `true`.
+the `action.destructive_requires_name` cluster setting must be `true`.
```

Comment:
Here and elsewhere, the doc earlier said about 'changing' the setting, so I think just an assertive statement would suffice.

Reply:
I think it's a little bit confusing that the first sentance says to use wildcards, and then the second sentance proceeds to explain that this is disallowed. I'd recommend rewording the whole paragraph as follows:

> By default, you must explicitly name the indices you are adding blocks to. To allow the adding of blocks to indices with `_all`, `*`, or other wildcard expressoins, change the `action.destructive_requires_name setting to `false`. You can update this setting...

Reply:
Done.